flake.nix

```diff
@@ -194,6 +194,7 @@
                 nodePackages.prettier
                 fd
                 git
+                gmp
```

Comment:
Why do we need `gmp`?

Reply:
Without `gmp` I was getting an error during the `stack build` stage: 

```bash
/nix/store/fzlkaj1ax7gl655blfcr6zzvml1vx3bj-binutils-2.40/bin/ld.gold: error: cannot find -lgmp
collect2: error: ld returned 1 exit status
`gcc' failed in phase `Linker'. (Exit code: 1)
```

Adding `gmp` to the dependencies solved this issue.

Reply:
Why do we use stack BTW?

Reply:
Decision after meeting: leave it as it is for now